src/Microsoft.DotNet.Tools.New/Program.cs

```diff
@@ -79,8 +99,21 @@ public static int Main(string[] args)
             app.Description = "Initializes empty project for .NET Platform";
             app.HelpOption("-h|--help");
 
+            var csharp = app.Option("-c#|--csharp", "Create an C# project", CommandOptionType.NoValue);
```

Comment:
I agree here, adding a new option for every language is going to be ugly eventually IMO.

I like `--lang` as there is a default (csharp) and it makes it clear this is an optional value.